packages/quip-cli/src/commands/login.ts

```diff
@@ -181,7 +187,11 @@ export default class Login extends Command {
         }
 
         try {
-            await login({ site, hostname, port, config });
+            if (accessToken) {
```

Comment:
@zhouhao If `with-token` is set but the access token is undefined/empty we should surface a message telling them so instead of redirecting them to the web login flow.

Reply:
Enabled now. Thank you